lms/lmsdb/models.py

```diff
@@ -93,6 +93,7 @@ class User(UserMixin, BaseModel):
     mail_address = CharField(unique=True)
     password = CharField()
     role = ForeignKeyField(Role, backref='users')
+    note = CharField(max_length=4096, default='')
```

Comment:
It should be many to one - each user should many notes (Notes -> User)

Reply:
This was my initial thinking but then I was thinking about our actual usage.
Wouldn't we want to have a general note for a specific user? or am I missing the usecase?